src/OpenSage.Game/Logic/Object/Behaviors/GenerateMinefieldBehavior.cs

```diff
@@ -33,6 +33,11 @@ internal override void Load(StatePersister reader)
             reader.PersistBoolean(ref _unknown);
 
             reader.SkipUnknownBytes(13);
+
+            if (reader.SageGame == SageGame.CncGeneralsZeroHour)
```

Comment:
```suggestion
            if (reader.SageGame >= SageGame.CncGeneralsZeroHour)
```
I guess there's not any way to know for _sure_ if this is correct, but we can assume?

Reply:
I can't remember whether any subsequent games used minefields 😄 If not, then I guess it's academic anyway.